### PR TITLE
feat(content): Conspiracy theorist spaceport news

### PR DESCRIPTION
**Pull request by tibetiroka**
**Content (Artwork / Missions / Jobs)**

This PR adds more spaceport news in topics requested on discord.

## Summary
More conspiracy theorist news about:
 - the Rigellian Plague
 - the Quarg
 - flat planets
 - diseases
 - hyperdrive tech
 - birds
 - drag

## Testing Done
I tested that the news show up and don't generate errors.

## Save File
N/A, pretty much any human planet is valid

### CODE REVIEW

File: data/human/news.txt [hunk 1]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
+		word
+			"Conspiracy Theorist"
+			"Free Thinker"
+			"Truth Seeker"
+	message
+		word
+			`"`
+		word
+			"The Rigellian Plague was a fake. The government used it to implant us with tracking microchips."
+			"If the Rigellian Plague really happened, how come we have so little quarantine facilities now?"
```
Comment (bene-dictator) [inline data/human/news.txt:1797]:
```suggestion
			"If the Rigellian Plague really happened, how come we have so few spaceport quarantine facilities now?"
```
Maybe they don't know the difference. It'd be funnier.

File: data/human/news.txt [hunk 2]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
+			"The Quarg built the pyramids of Earth. There is simply no way the people could have constructed something so massive back then."
+			"Spaceship windows are really just screens designed to show planets as round. One day, I will build my own ship to prove they are flat."
+			"The Quarg are not aliens, just actors in body suits. There isn't enough sunlight near gas giants for a species to evolve."
+			"Hyperdrives were given to the government by the Quarg. If it was our technology, we would have improved it by now."
+			"Aliens took our people to the Deep before hyperdrives were invented to experiment on them. This is how they created the Alphas."
+			"Republic Intelligence has bird-like surveillance drones in every city. There is one in front of my window every morning."
+			"The Quarg are running the government. There is no way such a powerful species wouldn't be able to control us."
+			"A friend of mine caught a horrible disease after visiting a Quarg ring. She had rashes and carbuncles all over her face."
+			"If spaceships have drag, there must be air in the 'vacuum of space'..."
```
Comment (bene-dictator) [inline data/human/news.txt:1806]: ~~Unrealistic.~~

File: data/human/news.txt [hunk 3]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
+			`"`
+		word
+			"The Rigellian Plague was a fake. The government used it to implant us with tracking microchips."
+			"If the Rigellian Plague really happened, how come we have so little quarantine facilities now?"
+			"The Quarg built the pyramids of Earth. There is simply no way the people could have constructed something so massive back then."
+			"Spaceship windows are really just screens designed to show planets as round. One day, I will build my own ship to prove they are flat."
+			"The Quarg are not aliens, just actors in body suits. There isn't enough sunlight near gas giants for a species to evolve."
+			"Hyperdrives were given to the government by the Quarg. If it was our technology, we would have improved it by now."
+			"Aliens took our people to the Deep before hyperdrives were invented to experiment on them. This is how they created the Alphas."
+			"Republic Intelligence has bird-like surveillance drones in every city. There is one in front of my window every morning."
```
Comment (roadrunner56) [inline data/human/news.txt:1803]:
```suggestion
			"Republic Intelligence has bird-like surveillance drones in every city. There's one in front of my window every morning."
```

File: data/human/news.txt [hunk 4]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
+			"The Quarg are running the government. There is no way such a powerful species wouldn't be able to control us."
```
Comment (roadrunner56) [inline data/human/news.txt:1804]:
```suggestion
			"The Quarg are running the government. There's no way such a powerful species wouldn't be able to control us."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]
Mostly just adding contractions to make some news lines sound more conversational.

File: data/human/news.txt [hunk 5]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
+			"Conspiracy Theorist"
+			"Free Thinker"
+			"Truth Seeker"
+	message
+		word
+			`"`
+		word
+			"The Rigellian Plague was a fake. The government used it to implant us with tracking microchips."
+			"If the Rigellian Plague really happened, how come we have so few spaceport quarantine facilities now?"
+			"The Quarg built the pyramids of Earth. There is simply no way that humans could have constructed something so massive back then."
```
[the remark this answers is not in the document]
Comment (tibetiroka) [reply data/human/news.txt:1798]: ~~But then I'd have to lock it behind FW, you see~~

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/human/news.txt:1798]: If anything I'd expect an actual alien invasion to bring way more conspiracy theories, yes :james:

[the remark this answers is not in the document]
Comment (tibetiroka) [reply data/human/news.txt:1798]: There are news for that in `pug news.txt`.

[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/human/news.txt:1798]: honestly conspiracy theorists knowing about the Pugs would be cool?
the deep know about them, aint hard to get those stories, and the name surely can be found if you dig a bit

**Review by roadrunner56 — APPROVED** [review]

File: data/human/news.txt [hunk 6]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
 
 
+news "conspiracy theorist"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Conspiracy Theorist"
+			"Free Thinker"
+			"Truth Seeker"
```
Comment (warp-core) [inline data/human/news.txt:1791]: News names only capitalise the first letter of the first word.
```suggestion
			"Conspiracy theorist"
			"Free thinker"
			"Truth seeker"
```

File: data/human/news.txt [hunk 7]
```diff
@@ -1781,6 +1781,41 @@ news "republic reporter in pirate space"
...
+			"The Quarg are running the government. There's no way such a powerful species wouldn't be able to control us."
+			"A friend of mine caught a horrible disease after visiting a Quarg ring. She had rashes and carbuncles all over her face."
+			"If spaceships have drag, there must be air in the 'vacuum of space'..."
+		word
+			"" 2
+			" Wake up, sheeple!"
+			" Do your own research!"
+			" If you still don't see the truth, I feel sorry for you."
+			" Leave the Matrix!"
+			" You are being lied to!"
```
Comment (Amazinite) [inline data/human/news.txt:1813]:
```suggestion
			" You are being lied to!"
			" Break the conditioning!"
```

**Pull request merged**